pkg/controllers/csi/metadata/sqlite.go

```diff
@@ -723,6 +730,47 @@ func (access *SqliteAccess) GetTenantsToDynakubes(ctx context.Context) (map[stri
 	return dynakubes, nil
 }
 
+func (access *SqliteAccess) GetAllAppMounts(ctx context.Context) []*Volume {
+	rows, err := access.conn.QueryContext(ctx, getAllAppMountsStatement)
+	if err != nil {
+		log.Info("skipping migration due to error getting all app mounts", "error", err)
+
+		return nil
+	}
+
+	defer func() { _ = rows.Close() }()
+
+	var volumes = make([]*Volume, 0)
+
+	for rows.Next() {
+		var code_module_version string
+
+		var volume_meta_id string
+
+		var location string
```

Comment:
`var code_module_version, volume_meta_id, location string`
you can combine these 3 lines into 1

Reply:
good one

Reply:
fixed in newest commit